src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -400,6 +401,7 @@ public void Initialize()
                     includeSubdirectories: false, changeTypes: WatcherChangeTypes.Created | WatcherChangeTypes.Changed);
 
                 _debugModeFileWatcher.Changed += OnDebugModeFileChanged;
+                TraceWriter.Verbose("Debug file watch initialized.");
```

Comment:
This one doesn't seem useful - this is a trivial operation. Seems like your goal here is to instead write traces for each major step in the init process?

Reply:
I'm seeing about 90 milliseconds of time between the previous log entry and this one, so I want to keep it. I think the threshold for a useless log entry should be < 5ms.